packages/p2p-v2/src/components/MobileTabs/MobileTabs.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import RightArrowIcon from '../../public/ic-chevron-right.svg';
+import './MobileTabs.scss';
+
+type TMobileTabsProps<T extends string[]> = {
+    onChangeTab: (clickedTab: T[number]) => void;
+    tabs: T;
+};
+
+function MobileTabs<T extends string[]>({ onChangeTab, tabs }: TMobileTabsProps<T>) {
```

Comment:
can't we use `Tabs` component here?

Reply:
These tabs are different and only for mobile:
<img width="480" alt="Screenshot 2024-01-29 at 6 17 36 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/d84e1199-a0be-47ea-8a85-7477bdcac763">